src/site/content/en/blog/gde-focus-nishu-goel/index.md

```diff
@@ -0,0 +1,68 @@
+
+**Monika**: Let's start with your story. What inspired you to become a developer and take on an active role within dev communities?
+
+**Nishu**: I got my bachelor's degree in computer science, we studied data structures, and that's where the interest in programming started. During the third year of engineering, a connection with the developer community was established through my participation in the Microsoft Imagine Cup Nationals competition where we presented solutions through code. The idea of the application we built was to bring educational opportunities to local students, especially girls. I met some very inspiring people, both contestants and organizers in this journey. 
+
+In 2018, my professional career took off, and I started working with Angular. Angular became the primary technology that connected me to the GDE program. Around the same time, I started writing blog posts and creating content around the subject I was working on and learning . Dhananjay Kumar helped me get started on this journey and ensured to keep me on track. My first articles tackled the basics of Angular. Soon after I started speaking at events-the first one being ngNepal, Nepal's Angular Conference. This led to more speaking invitations about Angular and web technologies. 
+
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/ujceg3BgclYOgIwDjtt7.jpeg", alt="Nishu and other people from ng-india.", width="800", height="533" %}
+
+**Monika**: What's your professional experience technology-wise?
+
+**Nishu**: It was all about Angular and web components for the first two years. I was using Angular for building the web, but soon I decided to go beyond that and explore other fields. I didn't want to limit myself in case I'd have to switch projects. That's how I started creating web components in Angular to use in other frameworks. 
+
+The first thing I did was to create web components using Angular. I published it to NPM and used it as a demo in a React project. I've discussed this during some of my talks and presentations later. My next job required using React and Typescript. Now, because I was working with React, I wasn't just using one framework anymore, but the web in general. At that moment I learned a lot about the web, especially web performance. That's when I had to start thinking about the [Largest Contenful Paint (LCP)](/lcp/) or [First Contentful Paint (FCP)](/fcp/), which means how much time it will take your application to load or what's going to be the maximum time for the page to render. I have been working towards choosing best practices  and an improved performance of the applications. 
```

Comment:
```suggestion
The first thing I did was to create web components using Angular. I published it to npm and used it as a demo in a React project. I've discussed this during some of my talks and presentations later. My next job required using React and Typescript. Now, because I was working with React, I wasn't just using one framework anymore, but the web in general. At that moment I learned a lot about the web, especially web performance. That's when I had to start thinking about the [Largest Contenful Paint (LCP)](/lcp/) or [First Contentful Paint (FCP)](/fcp/), which means how much time it will take your application to load or what's going to be the maximum time for the page to render. I have been working towards choosing best practices  and an improved performance of the applications. 
```

Reply:
npm should be in lowercase.